src/Pim/Bundle/ReferenceDataBundle/Doctrine/ORM/Repository/ReferenceDataRepository.php

```diff
@@ -90,6 +90,18 @@ public function findOneByIdentifier($identifier)
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function findByIdentifiers(array $referenceDataCodes)
```

Comment:
I'm not sure about this, maybe there is a lot of codes and we should only find them one by one ?